### PR TITLE
content update

### PR DESCRIPTION
**Pull request by BlladeRunner**
Adding styles and additional content

### CODE REVIEW

**Comment by BlladeRunner** [comment]
It is amazing..

File: index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+          <li>About Us</li>
+          |
+          <li>Contact Us</li>
+          |
+          <li>Block</li>
+          |
+          <li>Site Map</li>
+          |
+          <li>Eng</li>
+          &#11206;
```
Comment (sbadalov-grid) [inline index.html]: Use Css pseudo elements to add pipe sign "|" in navigation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          class="search"
+          type="search"
+          value
+          placeholder="Search this site"
+        />
+      </div>
+    </nav>
+
+    <!-- Hero section   -->
+    <div class="hero">
```
Comment (sbadalov-grid) [inline index.html]: for each section use 'section' semantic tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      <img
+        src="img/unsplash_W5XTTLpk1-I.svg"
+        alt="SVG Image"
+        width="500"
+        height="550"
+      />
+    </div>
+
+    <!-- ABOUT us section -->
+    <div class="about">
```
Comment (sbadalov-grid) [inline index.html]: section

File: index.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+      <div class="about-content2">
+        <img src="img/Rectangle 570.svg" alt="SVG Image" />
```
Comment (BlladeRunner) [inline index.html]: add width& height

File: index.html [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+        <h3>About Us</h3>
+        <span
```
Comment (BlladeRunner) [inline index.html]: better to use <p> is text

File: index.html [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+          hendrerit morbi sed gravida. Nulla aliquam ac diam amet, sagittis, ut
+          ultricies. Urna hendrerit morbi sed gravida. Lorem ipsum dolor sit
+          amet, consectetur adipiscing elit.</span
+        ><br />
+        <a href="">Learn more about &#8594;</a>
+      </div>
+      <div class="about-content3">
+        <img src="img/Rectangle 570.svg" alt="SVG Image" />
+        <h3>About Us 2</h3>
+        <span
```
Comment (BlladeRunner) [inline index.html]: better to use `<p>`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+          />
+        </div>
+      </nav>
+
+      <!-- Hero section   -->
+      <section class="hero">
+        <div class="hero-content">
+          <h1>Lorem ipsum dolor sit amet</h1>
+          <h3>Lorem ipsum dolor sit amet</h3>
+          <h5>
```
Comment (BlladeRunner) [inline index.html]: change from h5 to `<p>`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+            <a href="">Learn more about</a>
+          </div>
+        </div>
+        <div class="about-container">
+          <div class="about-img">
+            <img src="img/Rectangle 570.svg" alt="SVG Image" />
+          </div>
+          <div class="about-text">
+            <h3>About Us 2</h3>
+            <span
```
Comment (BlladeRunner) [inline index.html]: change `<span>` to p

File: index.html [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+              Urna hendrerit morbi sed gravida. Nulla aliquam ac diam amet,
+              sagittis, ut ultricies. Urna hendrerit morbi sed gravida. Lorem
+              ipsum dolor sit amet, consectetur adipiscing elit.
+            </p>
+            <a href="">Learn more about</a>
+          </div>
+        </div>
+        <div class="about-container">
+          <div class="about-img">
+            <img src="img/Rectangle 570.svg" alt="SVG Image" />
```
Comment (BlladeRunner) [inline index.html]: add meaningful `<alt>` text to every image on the page

File: index.html [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+            /></a>
+            <ul class="main-menu">
+              <li>About Us</li>
+              <li>Contact Us</li>
+              <li>Block</li>
+              <li>Site Map</li>
+              <li class="eng">Eng</li>
+            </ul>
+          </div>
+          <input
```
Comment (BlladeRunner) [inline index.html]: Add search button

File: index.html [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+            ><br />
+            <a href="">Learn more about</a>
+          </div>
+        </div>
+      </section>
+
+      <!-- CONTACT -->
+      <section class="contact">
+        <div class="container">
+          <div class="image">
```
Comment (BlladeRunner) [inline index.html]: change class name

File: index.html [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+            <h1>Contact Us</h1>
+            <p>
+              Fill up the form and our Team will get back to you within 24
+              hours. Happy to see your message!
+            </p>
+            <p class="fb-icon-vertic">
```
Comment (BlladeRunner) [inline index.html]: change p to address tag

File: index.html [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+                example@gmail.com</span
+              >
+              <img src="img/bi_facebook.svg" alt="SVG Image" /><span
+                class="fb-icon-text"
+              >
+                Pidmurma 5a, Lviv</span
+              >
+            </p>
+            <p class="fb-icon-horizon">
+              <img src="img/bi_facebook.svg" alt="SVG Image" />
```
Comment (BlladeRunner) [inline index.html]: add links

**Pull request merged**